framework/wazuh/core/batcher/mux_demux.py

```diff
@@ -0,0 +1,218 @@
+import uuid
+
+from multiprocessing import Queue, Process
+from multiprocessing.managers import DictProxy, SyncManager
+from typing import Optional
+
+
+class Message:
+    """
+    A message for the MuxDemuxQueue with an associated unique identifier.
+
+    Parameters
+    ----------
+    uid : uuid.UUID
+        The unique identifier for the message.
+    msg : dict
+        The message content as a dictionary.
+    """
+    def __init__(self, uid: uuid.UUID, msg: dict):
+        self.uid = uid
+        self.msg = msg
+
+
+class MuxDemuxQueue:
+    """
+    A queue for managing messages between mux and demux components.
+
+    Parameters
+    ----------
+    proxy_dict : DictProxy
+        A dictionary-like proxy for managing responses.
+    q_mux : Queue
+        The queue for multiplexing messages.
+    q_demux : Queue
+        The queue for demultiplexing messages.
+    """
+    def __init__(self, proxy_dict: DictProxy, q_mux: Queue, q_demux: Queue):
+        self.responses = proxy_dict
+        self.q_mux = q_mux
+        self.q_demux = q_demux
```

Comment:
```suggestion
    mux_queue: Queue
        The queue for multiplexing messages.
    demux_queue: Queue
        The queue for demultiplexing messages.
    """
    def __init__(self, proxy_dict: DictProxy, mux_queue: Queue, demux_queue: Queue):
        self.responses = proxy_dict
        self.mux_queue = mux_queue
        self.demux_queue = demux_queue
```

Reply:
Fixed in https://github.com/wazuh/wazuh/commit/47e4508357e0fb53f499847d6a791a0f4c86f15b.